lib/Mojo/mysql/Migrations.pm

```diff
@@ -28,204 +22,150 @@ sub from_string {
   }
-
   return $self;
 }
-
 sub latest { (sort keys %{shift->{migrations}{up}})[-1] || 0 }
 
+
 sub migrate {
   my ($self, $target) = @_;
   $target //= $self->latest;
 
-  # Unknown version
+# Unknown version
   my ($up, $down) = @{$self->{migrations}}{qw(up down)};
   croak "Version $target has no migration" if $target != 0 && !$up->{$target};
 
-  # Already the right version (make sure migrations table exists)
+# Already the right version (make sure migrations table exists)
   my $db = $self->mysql->db;
   return $self if $self->_active($db) == $target;
 
-  # Lock migrations table and check version again
+# Lock migrations table and check version again
   local @{$db->dbh}{qw(AutoCommit RaiseError)} = (1, 1);
   my $tx = $db->begin;
 
   return $self if (my $active = $self->_active($db)) == $target;
 
-  # Up
+# Up
   my $sql;
   if ($active < $target) {
-    $sql = join '',
-      map { $up->{$_} } grep { $_ <= $target && $_ > $active } sort keys %$up;
+    my @up = grep { $_ <= $target && $_ > $active } sort keys %$up;
+    $sql = join '', map { $up->{$_} } @up;
   }
 
-  # Down
+# Down
   else {
-    $sql = join '',
-      map { $down->{$_} }
-      grep { $_ > $target && $_ <= $active } reverse sort keys %$down;
+    my @down = grep { $_ > $target && $_ <= $active } reverse sort keys %$down;
+    $sql = join '', map { $down->{$_} } @down;
   }
-
   warn "-- Migrate ($active -> $target)\n$sql\n" if DEBUG;
-
   chop($sql);
-  $db->query($_) for split(';', $sql); 
-  $sql = "update mojo_migrations set version = ? where name = ?;";
+  $db->query($_) for split(';', $sql);
+  $sql = 'update mojo_migrations set version = ? where name = ?;';
   $db->query($sql, $target, $self->name) and $tx->commit;
   return $self;
 }
 
 sub _active {
   my ($self, $db) = @_;
-
   my $dbh  = $db->dbh;
   my $name = $self->name;
   local @$dbh{qw(AutoCommit RaiseError)} = (1, 0);
-  my $results
-    = $db->query('select version from mojo_migrations where name = ?', $name);
+  my $results = $db->query('select version from mojo_migrations where name = ?', $name);
   if (my $next = $results->array) { return $next->[0] }
-
   local @$dbh{qw(AutoCommit RaiseError)} = (1, 1);
   $db->query(
     'create table if not exists mojo_migrations (
        name    varchar(255) unique,
-       version varchar(255)
-     )'
+       version varchar(255) not null check (version >= 0)
+     ) ENGINE=InnoDB'
   ) if $results->sth->err;
   $db->query('insert into mojo_migrations values (?, ?)', $name, 0);
-
   return 0;
 }
-
 1;
 
 =encoding utf8
-
 =head1 NAME
-
 Mojo::mysql::Migrations - Migrations
-
 =head1 SYNOPSIS
-
-  use Mojo::mysql::Migrations;
-
-  my $migrations = Mojo::mysql::Migrations->new(mysql => $mysql);
-  $migrations->from_file('/Users/sri/migrations.sql')->migrate;
-
+use Mojo::mysql::Migrations;
+my $migrations = Mojo::mysql::Migrations->new(mysql => $mysql);
+$migrations->from_file('/Users/sri/migrations.sql')->migrate;
```

Comment:
This POD formatting looks very strange. I would suppose this will be rendered on one line.